From 1263264b5af268dba19f27520a7fb135a5c8f07e Mon Sep 17 00:00:00 2001
From: Yichong Chen <chenyichong@uniontech.com>
Date: Thu, 18 Jun 2026 18:02:54 +0800
Subject: [PATCH 10374/15333] tools/virtio: Remove unsupported --batch option
 from vhost_net_test
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

vhost_net_test has --batch in longopts, but not in help.

The parser never handles 'b', so --batch hits assert(0).

Remove the unsupported option.

Signed-off-by: Yichong Chen <chenyichong@uniontech.com>
Acked-by: Eugenio Pérez <eperezma@redhat.com>
Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Message-ID: <E091F15D8FBE8F16+20260618100254.513806-1-chenyichong@uniontech.com>
---
 tools/virtio/vhost_net_test.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/tools/virtio/vhost_net_test.c b/tools/virtio/vhost_net_test.c
index 389d99a6d7c7..566e15420bb6 100644
--- a/tools/virtio/vhost_net_test.c
+++ b/tools/virtio/vhost_net_test.c
@@ -450,11 +450,6 @@ static const struct option longopts[] = {
 		.val = 'n',
 		.has_arg = required_argument,
 	},
-	{
-		.name = "batch",
-		.val = 'b',
-		.has_arg = required_argument,
-	},
 	{
 	}
 };
-- 
2.55.0

